src/constructs/ec2/vpc.ts

```diff
@@ -16,19 +18,26 @@ export enum SubnetType {
 
 export interface GuSubnetProps {
   type?: SubnetType;
+  app?: string;
```

Comment:
Could `GuSubnetProps` extend [`AppIdentity`](https://github.com/guardian/cdk/blob/d69bdbe181d90f0f2c502dee9b05095f1402301a/src/constructs/core/identity.ts#L9-L11) instead?

Reply:
As discussed IRL, this is not a trivial change and so we've elected to leave this for a future PR.